src/components/MapTiff/MapTiff.styles.tsx

```diff
@@ -0,0 +1,6 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  width: 100vw;
+  height: 100vh;
```

Comment:
We want to use this components in another pages and places. Because of it's better if we make the width and height of the component more flexible.

```suggestion
  width: 100%;
  height: 100%;
```